src/mixins/filing-template-mixin.ts

```diff
@@ -313,25 +313,23 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
         folioNumber: this.getFolioNumber,
         isFutureEffective: this.getEffectiveDateTime.isFutureEffective
       },
-      business: {
-        legalType: this.getEntityType,
-        identifier: this.getTempId
-      },
       registration: {
         startDate: this.getRegistration.startDate,
         nameRequest: {
           legalType: this.getEntityType
         },
-        nameTranslations: this.getNameTranslations,
-        offices: {},
```

Comment:
Why are you removing `nameTranslations` and `offices` objects?

And why are you saving an empty `businessAddress` object?

Reply:
Ah, OK, that was left over from the initial template.